configs/projects/_shared-agent-settings.rb

```diff
@@ -176,6 +176,14 @@
   proj.setting(:ldflags, "-L#{proj.libdir} -Wl,-rpath=#{proj.libdir},-z,relro,-z,now")
 end
 
+if platform.name =~ /^redhatfips-/
+  proj.setting(:openssl_version, '1.1.1-fips')
```

Comment:
Previously the `openssl_version` setting was `1.1.1` on FIPS, which caused it to be installed via yum due to the `pkg.build_requires` statements. The vanagon logic first attempts to search for an included component (such as `openssl-1.1.1.rb` which is not used for RedHat FIPS), then falls back to installing the dependency with `yum`.